src/components/Table/GridTable.stories.tsx

```diff
@@ -526,13 +526,13 @@ export const DataTypeColumns = newStory(
     const priceCol = numericColumn<Row2>({
       header: "Price",
       data: ({ priceInCents }) => (
-        <NumberField hideLabel label="Price" value={priceInCents} onChange={noop} type="cents" />
+        <NumberField labelStyle="hidden" label="Price" value={priceInCents} onChange={noop} type="cents" />
```

Comment:
**FWIW:** We could just remove this property all together. `GridTable` has a `<PresentationProvider />` wrapped around it that should set the `labelStyle: "hidden"` accordingly. That should allow any field which reads this property to inherit it properly from the PresentationContext.